api/hasura/actions/_handlers/updateAllocations.ts

```diff
@@ -133,6 +133,22 @@ async function handler(req: VercelRequest, res: VercelResponse) {
     const existingGift = pending_sent_gifts.find(
       g => g.recipient_id === gift.recipient_id
     );
+
+    ops['insertInteractionEvents'] = {
```

Comment:
more verbose to add it here, but its wrapped into one sql mutation vs if we used the insertInteractionEvents call...